### ISSUE TITLE
Ubuntu 24.04 LTS update any time soon?

### ISSUE THREAD

**Issue by ricperry** [TheAssassin/AppImageLauncher#641]
### Pre-submit checks

- [X] I checked for similar issues beforehand, but could not find any. I could not add my proposal to any existing issue.
- [X] I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

### Feature description

I just want to be able to "install" appimages to the appropriate location and added to the launcher menus without the hassle of manually placing and editing files myself. Is this going to be coming to Ubuntu 24.04 LTS any time soon?

### Proposed solution

Package the build with required dependencies for Gnome 46 / Ubuntu 24.04 LTS

### Alternative solutions you considered

I'm not aware of any other than laboriously installing individual appimages myself.

**Comment by InklingGirl** [comment]
+1

**Comment by erkinalp** [comment]
depends on #634

**Comment by Munk3y** [comment]
+1

**Comment by TheAssassin** [comment]
The `.deb` from the [release page](https://github.com/TheAssassin/AppImageLauncher/releases/tag/continuous) works fine on Ubuntuu 24.04. I use it every day, on multiple systems.

Before you ask: the PPA is broken and I don't think a fix is on the way. That's tracked in a separate issue.

**Comment by ricperry** [comment]
Thanks. I wasn't aware of the release page, and was just trying to follow the instructions from the main page of this git.
![Screenshot from 2024-05-06 12-57-25](https://github.com/TheAssassin/AppImageLauncher/assets/34995372/3820e9ad-4a1f-4f5d-9d2d-da8ec2ca91f3)

**Comment by TheAssassin** [comment]
The README is a little out of date. I still have hopes for the PPA, but right now, it's broken.
